src/nunavut/cli.py

```diff
@@ -179,13 +184,23 @@ def _build_post_processor_list_from_args(args: argparse.Namespace) \
                     sys.stdout.write(';')
         return 0
 
-    if _should_generate_support(args):
+    if _should_generate_support(args, language_context):
         support_generator.generate_all(is_dryrun=args.dry_run,
                                        allow_overwrite=not args.no_overwrite)
 
     if args.generate_support != 'only':
         generator.generate_all(is_dryrun=args.dry_run,
                                allow_overwrite=not args.no_overwrite)
+
+    # If docgen selected, warn about linked namespaces
+    if args.target_language == 'docgen':
+        if len(extra_includes) > 0:
+            print(textwrap.dedent("""
+                W: Other lookup namespaces are linked in these generated docs.
+                If you do not generate docs for these other namespaces as well, links to external
+                data types could be broken (expansion will still work).
+            """).lstrip())
```

Comment:
Did you mean to use `logging.warning` here?

Reply:
Oh, sorry. I did intend to use `logging.warning` here.

All that warning means is that if the Namespace.html links to other namespaces (i.e. `--lookup`), that namespace needs to be separately generated. Perhaps I can make that wording a bit more clear.